.travis.yml

```diff
@@ -1,14 +1,19 @@
 language: java
 sudo: false
 script: 
-  - git clone https://github.com/SupunArunoda/CPACEP.git
+  - cd ..
+  - cd ..
+  - cd CPACEP
+  - mkdir /tmp/test
   - mvn clean verify
-  - mvn test
+  - ls /tmp/test
   
-
-before_script:
+before_install:
+  - cd ..
+  - mkdir CPAChecker
+  - cd CPAChecker
   - git clone https://github.com/sosy-lab/cpachecker.git
+  - mkdir /tmp/original
   - cd cpachecker
   - ant build
```

Comment:
I already mentioned this by mail, but to have everything in one place: CPAchecker is not necessary for all build steps, so I would only install it for jobs that actually require it.